test/client/stringify.spec.js

```diff
@@ -28,59 +27,62 @@ describe('stringify', function () {
     var partsDef = ['function', '(d, e, f)', '{ ... }']
 
     partsAbc.forEach(function (part) {
-      expect(abcString).to.contain(part)
+      assert(abcString.indexOf(part) > -1)
     })
 
     var defString = stringify(def)
     partsDef.forEach(function (part) {
-      expect(defString).to.contain(part)
+      assert(defString.indexOf(part) > -1)
     })
   })
 
   it('should serialize arrays', function () {
-    expect(stringify(['a', 'b', null, true, false])).to.be.eql("['a', 'b', null, true, false]")
+    assert.deepEqual(stringify(['a', 'b', null, true, false]), "['a', 'b', null, true, false]")
   })
 
   it('should serialize objects', function () {
     var obj
 
     obj = {a: 'a', b: 'b', c: null, d: true, e: false}
-    expect(stringify(obj)).to.contain("{a: 'a', b: 'b', c: null, d: true, e: false}")
+    assert(stringify(obj).indexOf("{a: 'a', b: 'b', c: null, d: true, e: false}") > -1)
 
     function MyObj () {
       this.a = 'a'
     }
 
     obj = new MyObj()
-    expect(stringify(obj)).to.contain("{a: 'a'}")
+    assert(stringify(obj).indexOf("{a: 'a'}") > -1)
 
     obj = {constructor: null}
-    expect(stringify(obj)).to.contain('{constructor: null}')
+    assert(stringify(obj).indexOf('{constructor: null}') > -1)
 
     obj = Object.create(null)
     obj.a = 'a'
-    expect(stringify(obj)).to.contain("{a: 'a'}")
+    assert(stringify(obj).indexOf("{a: 'a'}") > -1)
   })
 
   it('should serialize html', function () {
     var div = document.createElement('div')
 
-    expect(stringify(div)).to.be.eql('<div></div>')
+    assert.deepEqual(stringify(div).trim(), '<div></div>')
 
     div.innerHTML = 'some <span>text</span>'
-    expect(stringify(div)).to.be.eql('<div>some <span>text</span></div>')
+    assert.deepEqual(stringify(div), '<div>some <span>text</span></div>')
   })
 
   it('should serialize DOMParser objects', function () {
-    var parser = new DOMParser()
-    var doc = parser.parseFromString('<test></test>', 'application/xml')
-    expect(stringify(doc)).to.be.eql('<test></test>')
+    if (DOMParser) {
+      // Test only works in IE 9 and above
+      var parser = new DOMParser()
+      var doc = parser.parseFromString('<test></test>', 'application/xml')
+      assert.deepEqual(stringify(doc), '<test></test>')
+    }
   })
 
   it('should serialize across iframes', function () {
     var div = document.createElement('div')
-    expect(__karma__.stringify(div)).to.be.eql('<div></div>')
+    assert.deepEqual(__karma__.stringify(div).trim(), '<div></div>')
```

Comment:
You need such a change in one of upper tests as well (the `should serialize html` one)